### PR TITLE
Allow local uploads

### PR DESCRIPTION
**Pull request by biancarosa**
# Description

This pull-request allows local uploads by changing the interface during local development if the LOCAL=TRUE environment variable exists.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [x] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I run `go fmt ./...` to check that my code is properly formatted
- [x] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: pkg/services/files/uploader.go [hunk 1]
```diff
@@ -49,16 +50,22 @@ type FileUploader struct {
...
 }
 
-// UploadFile is Basically a dummy function that returns no error but allows offline
-// development without S3 and satisfies the interface
+// UploadFile basically copies a file to the local server path
+// Allowing offline development without S3 and satisfying the interface
 func (u *FileUploader) UploadFile(fname string, uploadPath string) (string, error) {
-	return fname, nil
+	destfile := u.BaseDir + "image.iso"
+	cmd := exec.Command("cp", fname, destfile)
```
Comment (AaronH88) [inline pkg/services/files/uploader.go:63]: Great idea! Nice and simple, good work :grin:

**Review by acosferreira — APPROVED** [review]
LGTM

File: pkg/services/files/uploader.go (hunk 1 above)
Comment (maxamillion) [reply pkg/services/files/uploader.go:63]: Should this use golang `io.Copy()` instead of `cp` command?

Comment (Avielyo10) [reply pkg/services/files/uploader.go:63]: `io.CopyN()` is preferred security-wise  (avoid potential DoS vulnerability via decompression bomb)

**Comment by Avielyo10** [comment]
Can we have more tests on this one?

**Pull request merged**